src/widget.ts

```diff
@@ -1,16 +1,19 @@
-import { DOMWidgetView } from "@jupyter-widgets/base";
+import { DOMWidgetView, DOMWidgetModel } from "@jupyter-widgets/base";
 import { vegaEmbed } from "./index";
 import { Result } from "vega-embed";
+import * as ndarray from "ndarray";
+import { table_serialization, rowProxy, IDict } from "jupyter-tablewidgets";
 
 interface WidgetUpdate {
   key: string;
   remove?: string;
-  insert?: any[];
+  insert?: any[] | string;
 }
 
 interface WidgetUpdateMessage {
   type: "update";
   updates: WidgetUpdate[];
+  resize: boolean;
```

Comment:
Please clarify whether this is needed or whether the existing autosize property in Vega is enough.

Reply:
I suspect it's better to explicitly call resize as you do here but I want to make sure you agree.